contact.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+$nameErr = $priceErr = $minCountErr = $maxCountErr = "";
+$name = $price = $playerCountMin = $playerCountMax = $image = $gameType = "";
+
+if (
+    isset($_POST['name']) && isset($_POST['price']) && isset($_POST['playerCountMin'])
+    && isset($_POST['playerCountMax']) && isset($_POST['image']) && isset($_POST['gameType'])
+) {
+
+    $name = $_POST['name'];
+    $price = $_POST['price'];
+    $playerCountMin = $_POST['playerCountMin'];
+    $playerCountMax = $_POST['playerCountMax'];
+    $image = $_POST['image'];
+    $gameType = $_POST['gameType'];
+
+    if ($_SERVER["REQUEST_METHOD"] == "POST") {
+
+        if (empty($_POST["name"])) {
+            $nameErr = "Name is required";
+        } else {
+            $name = ($_POST["name"]);
+        }
+        if (empty($_POST["price"])) {
+            $priceErr = "RRP is required";
+        } else {
+            $price = ($_POST["price"]);
+        }
+        if (empty($_POST["playerCountMin"])) {
+            $minCountErr = "Minimum no. of players is required";
+        } else {
+            $playerCountMin = ($_POST["playerCountMin"]);
+        }
+        if (empty($_POST["playerCountMax"])) {
+            $maxCountErr = "Maximum no. of players is required";
+        } else {
+            $playerCountMax = ($_POST["playerCountMax"]);
+        }
+        //check no error messages - all fields are complete: 
+        //if none then submit to database and move to homepage - still need to write code to send to db!
+
+        if ($nameErr == "" && $priceErr == "" && $minCountErr == "" && $maxCountErr == "") {
+            header('Location: index.php');
+        }
+
+        $db = new PDO('mysql:host=db; dbname=collections_display', 'root', 'password');
+
+        $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+        $query = $db->prepare("INSERT INTO board_games (`name`, `rrp`, `min_player_count`, `max_player_count`, `image`, `type`) 
+            VALUES ('$name', '$price', '$playerCountMin', '$playerCountMax', '$image', '$gameType')");
+
+        $query->execute();
+    }
+}
+
+
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+    <style>
```

Comment:
Now that you have it all working, you should move this CSS into a a CSS file before this goes to main

Reply:
Done